mdagent/tools/maketools.py

```diff
@@ -74,6 +92,24 @@ def make_all_tools(
         SmallMolPDB(path_registry=path_instance),
```

Comment:
The list of tools is starting to get big, should we include a way for the user to select which tools he wants? 
Like a list of tools strings, even for us, when testing could make it easier, as we wouldnt need to come here and comment out this part of the code

Reply:
Yeah, we can just make it an init option when initializing mdagent, specifying which tools or all